everything_else/reproducing.m

```diff
@@ -157,6 +157,81 @@
 %% Pg. 21, PTR-BCR-2
 %% Pg. 22, PTR-BCR-3 (example appears to be the same as PTR-BCR-1, and may have to be redone)
 %% Pg. 23, PTR-BCR-4 
+
+% Eq. 72
+b = dec2bin(2^6-1:-1:0)-'0';
+b1=b(:,1);b2=b(:,2);b3=b(:,3);b4=b(:,4);ba1=b(:,5);ba2=b(:,6);
+LHS = min(reshape(b1.*b2.*b3.*b4, 4, []));
+RHS = min(reshape((b1 + b2 + b3 + b4 - ba1 - 2*ba2).^2, 4, []));
+isequal(LHS,RHS); % Gives 1, confirmed by Nike on 6 April.
+
```

Comment:
While I appreciate that it would take a long time to type out `b1*b2 + b1*b3 + b1*b4 + ...` for all terms, I think that it would be more "readable". For all other quadratizations verified in this file so far, it's easy to compare the equation in the book with the code and immediately see that they're the same. In this case, it's a lot more complicated. Perhaps you can use MATLAB's symbolic computing toolbox to help. **I think I'd prefer MATLAB code that looks as close as possible to the equations in the book, then we can _also_ keep your code containing the `for` loops (so we have two different verifications for each equation: one "long" one that's easy to read, and one "compact" one which requires the reader to interpret a bunch of `for` and `end` statements).**